contrib/babelfishpg_tsql/src/session.c

```diff
@@ -15,12 +16,26 @@
 
 /* Core Session Properties */
 
+#define MAX_SYSNAME_LEN 512 /* Large enough to handle 128 character unicode strings */
```

Comment:
This is sort of a magic number here, is there maybe a better way to get this value?

Reply:
no at the moment